pom.xml

```diff
@@ -5,7 +5,7 @@
     <name>Signature API-client Java</name>
     <groupId>no.digipost.signature</groupId>
     <artifactId>signature-api-client-java</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>0.2-SNAPSHOT</version>
```

Comment:
Yes, I agree.
I guess I wanted to indicate that it probably should be released as version 0.2 before merging to master.